site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,136 @@
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/variables.css' lang:css %}
+
+We can now use these variables in `worker.m.css` to create our fully themed `worker`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/worker.m.css' lang:css %}
+
+{% section %}
+
+## Theming Dojo widgets
+
+So far in this tutorial we have themed our custom `worker` widget, but how do we target dojo widgets with our theme. To demonstrate this, we will theme the `workerForm` widget as it contains both domNodes and dojo widgets to be themed.
+
+{% instruction 'Lets create `workerForm.m.css` and include it in our theme.ts' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/workerForm.m.css' lang:css %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,5,9,11,13,16 %}
+
+This should be familiar to you by now as we covered theming `worker` in the last section. Now to theme the Dojo 2 `TextInput` that `WorkerForm` uses, we will need to create a `dojoTextInput.m.css` file and export it from `theme.ts` using a special theme key which is prefixed with `dojo-`. This prefix helps to ensure that our own widget theme files do not clash with dojo specific widgets and makes it easy to recognise when we are targeting one of the other.
+
+{% instruction 'Create `dojoTextInput.m.css` and emport it from `theme.ts`' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/dojoTextInput.m.css' lang:css %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/themes/dojo/theme.ts' lines:3,5,6,9,11,13,14,16 %}
+
+In this example we are introducing another powerful part of the Dojo 2 theming system, `composes`. Composes comes from the `css-module` specification and allows you to take styles from one class selector and apply them to another. In this case we are saying that `root` of a `TextInput` (the label text in this case), should look the same as the `nameLabel` class in our `WidgetForm`. This approach can be very useful when creating multiple themes from a baseTheme and avoids re-writing / re-specifying styles in multiple places.
+
+If you open up your web browser now you should see that the Text Inputs at the top of the form have been styled.
+
+{% instruction 'Why dont you go ahead and create a theme file for dojo button' %}
+
+Have a go at creating a theme file for the Dojo button thats used on `WidgetForm`. If you need help you can see the finished `dojoButton.m.css` file [here](demo/finished/biz-e-corp/src/themes/dojo/dojoButton.m.css).
+
+{% section %}
+
+## Summary
+
+In this tutorial, we learned how to create a theme and apply a theme to both Dojo and custom widgets using the themeInjector. We learnt how to use some of the functionality of the Dojo 2 theming system to apply variables and to use `composes` to share styles between components.
+
+If you would like to you can continue to work with the theme we have started here to tweak and change things as you please.
```

Comment:
If you would like to you can continue to work with the theme we have started here to tweak and change things as you please.  .... I would probably just cut this sentence